### PR TITLE
⬆️ Bump starlette to 0.28.0

### PR DESCRIPTION
**Pull request by Kludex**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -41,7 +41,7 @@ classifiers = [
     "Topic :: Internet :: WWW/HTTP",
 ]
 dependencies = [
-    "starlette>=0.27.0,<0.28.0",
+    "starlette>=0.27.0,<0.29.0",
```
Comment (Kludex) [inline pyproject.toml:44]: We didn't add any breaking change, but change the range as you wish. 👀

Comment (Kludex) [reply pyproject.toml:44]: nevermind, it looks like starlette breaks fastapi test suite.

**Comment by huaiguoguo** [comment]
when will "starlette" be updated to latest version 
latest version of starlette is  0.28.0

**Comment by Kludex** [comment]
Subscribe to https://github.com/tiangolo/fastapi/pull/9636.

**Pull request closed without merging**